code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -252,27 +248,23 @@
 	taste_description = "salt" // because they're bathsalts?
 	var/datum/brain_trauma/special/psychotic_brawling/bath_salts/rage
 
-/datum/reagent/drug/bath_salts/on_mob_add(mob/M)
+/datum/reagent/drug/bath_salts/on_mob_add(mob/living/L)
 	..()
-	if(isliving(M))
-		var/mob/living/L = M
-		L.add_trait(TRAIT_STUNIMMUNE, id)
-		L.add_trait(TRAIT_SLEEPIMMUNE, id)
-		if(iscarbon(L))
-			var/mob/living/carbon/C = L
-			rage = new()
-			C.gain_trauma(rage, TRAUMA_RESILIENCE_ABSOLUTE)
+	L.add_trait(TRAIT_STUNIMMUNE, id)
+	L.add_trait(TRAIT_SLEEPIMMUNE, id)
+	if(iscarbon(L))
```

Comment:
Might as well get rid of this ?

Reply:
`on_mob_add()` works on the living, not just carbons